readme.md

```diff
@@ -53,8 +53,14 @@ const execa = require('execa');
 	//=> 'unicorns'
 
 	// Cancelling a spawned process
-	const spawned = execa("echo unicorns");
+	const spawned = execa("node");
 	spawned.cancel();
```

Comment:
Should we put this in a `setTimeout(() => { spawned.cancel() }, 1000)` block? So that it's clear the process can be canceled after being `await`ed?